packages/@stylexjs/eslint-plugin/__tests__/stylex-valid-styles-test.js

```diff
@@ -1857,5 +1857,89 @@ revert`,
         },
       ],
     },
+    {
+      code: `
+        import * as stylex from '@stylexjs/stylex';
+        const styles = stylex.create({
+          invalidStyle: {
+            margin: "10",
+            height: "10",
+          },
+        });
+      `,
+      errors: [
+        {
+          message:
+            'margin value must be one of:\n' +
+            'a number literal or math expression\n' +
+            'a non-numeric string\n' +
+            'null\n' +
+            'initial\n' +
+            'inherit\n' +
+            'unset\n' +
+            'revert',
+        },
+        {
+          message:
+            'height value must be one of:\n' +
+            'a non-numeric string\n' +
+            'a number literal or math expression\n' +
+            'available\n' +
+            'min-content\n' +
+            'max-content\n' +
+            'fit-content\n' +
+            'auto\n' +
+            'a number ending in px, mm, in, pc, pt\n' +
```

Comment:
Curious why this message is present for height but not margin? Aligning on official syntax might help with consistency